src/neo/SmartContract/ContractParametersContext.cs

```diff
@@ -325,6 +332,7 @@ public JObject ToJson()
             json["items"] = new JObject();
             foreach (var item in ContextItems)
                 json["items"][item.Key.ToString()] = item.Value.ToJson();
+            json["magic"] = Magic;
```

Comment:
Not critical, but it's called `network` in neo-go already.

Reply:
fixed